packages/mongodb-chatbot-ui/src/useTextInputTrigger.ts

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react";
+import { MongoDbInputBarPlaceholder } from "./InputBar";
+import { defaultChatbotFatalErrorMessage } from "./ui-text";
+import { useChatbotContext } from "./useChatbotContext";
+import { ChatbotTriggerProps } from "./ChatbotTrigger";
+
+export type UseTextInputTriggerArgs = {
+  placeholder?: string;
+  fatalErrorMessage?: string;
+};
+
+/**
+  The base props for a Chatbot trigger component that allows the user to input text.
+ */
+export type ChatbotTextInputTriggerProps = ChatbotTriggerProps &
```

Comment:
do we need to update the public facing docs for this PR?

Reply:
Nothing required, though we could cover custom triggers/view more generally in the docs & mention this there.

https://jira.mongodb.org/browse/EAI-468